internal/validation/validation_test.go

```diff
@@ -25,6 +26,18 @@ type Test struct {
 }
 
 func TestValidate(t *testing.T) {
+	skip := map[string]struct{}{
+		// The meta schema always includes the standard types, so this isn't applicable
+		"Validate: Known type names/references to standard scalars that are missing in schema": {},
+		// Ignore tests using experimental @stream
+		"Validate: Overlapping fields can be merged/different stream directive label":               {},
+		"Validate: Overlapping fields can be merged/different stream directive initialCount":        {},
+		"Validate: Overlapping fields can be merged/different stream directive first missing args":  {},
+		"Validate: Overlapping fields can be merged/different stream directive second missing args": {},
+		"Validate: Overlapping fields can be merged/different stream directive extra argument":      {},
+		"Validate: Overlapping fields can be merged/mix of stream and no stream":                    {},
+	}
```

Comment:
It is awesome that only the stream tests are being skipped. This is expected for now.

Reply:
Not quite as good as this might suggest... check `export.ts`. I've maintained parity with the old implementation there in terms of which tests are imported (which get added to the `tests.json`). Many test files had been excluded, and I believe additional ones have also been added to `graphql-js` since, which also aren't being imported here.

I expect getting those to behave correctly and pass the tests is going to quite a bit more work, so have left that out of this change, but will start having a go at that afterwards.